src/middlewares/errorHandler/ErrorHandlerMiddleware.ts

```diff
@@ -6,7 +6,7 @@ const ErrorHandler: ErrorRequestHandler = (
   res: Response,
   next: NextFunction
 ) => {
-  res.status(500).send({ err });
+  res.send({ err });
```

Comment:
this middleware is unused i think, do we even need it 😢

Reply:
Nah, forgot to remove lol.

Reply:
Removed, also removed validation middleware bc we use class-validator now

Reply:
thanks!